clusterloader2/clusterloader.go

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Please move this to cmd/clusterloader.go

Reply:
Code structure has been changed:
- clusterloader.go moved to cmd/
- all other directories (except api and vendor) moved to pkg/